src/helpers/ramlParser.js

```diff
@@ -103,14 +103,14 @@ function _parseRAML(filePath, dest, baseUri, listTraits, cb) {
     creator.createFilesSync(dest, result, 'utf-8');
     return cb('Generation went fine');
   }, (err) => {
-    if(err && err.message && err.message.indexOf('ECONNREFUSED') !== -1) {
-      logger.warning('Could not download traits.');
-      return cb();
+    if(err && err.message && err.message.indexOf('ECONNREFUSED') !== -1 || err && err.message && err.message.indexOf('ETIMEDOUT') !== -1) {
+      logger.error(`Could not download traits for: ${dest}`);
+      return process.exit(1);
```

Comment:
AFAIK this `cb()` that was here previously canceled generation as well in the past, why do we need to force it?